# Working Title: The role Gene Expression in Measuring/Establishing/Determining Local Adaptation

## Thoughts

Driving Question: How can gene expression be used to study local adaptation?

Can study SNPs, but there is a lot of other points of regulation that can impact the final gene product.  
Use eQTLs to narrow down causal sequence variation.  
What information does only looking at genetic sequence and/or morpholocial traits miss?  
How can gene expression bolster/support/supplement that information?

### Important Concepts to define

* local adaptation
  * a population has higher fitness in it's local environment compared to another population
  * this increase in fitness comes at the cost of fitness in other environments
  * Savolainen et al.
* gene expression (? maybe I don't need to define, but I think I do need to define the scope and what I am specifically discussing)

### Why study local adaptation at all?

Species/populations are encountering novel environments, sometimes stressful
  * some species/populations have adapted to those new environments - morphological, phenology

By better understanding the genetic mechanisms of this change, we can better _predict_ how species will change in certain environments.

This is important to know for both agricultural systems, as humans breed or modify crops to meet both rising demand and stressful environments, and for biodiversity conservation of undomesticated flora and fauna.

## Sections

### Background on local adaptation

Intended to be short, as this is extensively covered in other reviews and papers

#### How local adaptation is studied

* Savolainen et al.
  * molecular methods
    * QTL mapping
    * Population genetics
    * association mapping
  * data
    * genetic sequence data - resequencing, whole genome, exome
    * reference genomes
    * trait phenotypes

* Bradshaw et al. - haven't yet read.
  * evolution of local adaptation due to climate change-mediated selection

* Experimental designs to study local adaptation:
  * common gardens
  * reciprocal transplant experiments (subset of common garden)

* Models of Local Adaptation
  * continuum
  * island

Looking for phenotypic variation along environmental clines


- genetic information to get population genetics - Fst
- morphological traits - Qst

#### Causes of local adaptation

Situations that local adaptation is likely to occur - Savolainen et al.

### Remaining unknowns about local adaptation

* the genetic architecture and the mechanisms underlying local adaptation - Savolainen et al.
* why that's hard: need more statistical power, either by increasing population size or including meaningful biological data
* some ways to do that are: increase sample population size/experiment size, improving models
* Another way is to combine genomic data and gene expression data

### Role of gene expression

Which questions about local adaptation can studying gene expression answer?

* 

* role of gene expression in speciation - Pavey et al. 
* mis-regulation as adaptive - Baym et al.
* gene expression variation and traits - Albert and Kruglyak

#### Tools used to measure gene expression

* RNA-Seq
* ATAC-Seq
* Methylation Sequencing?

* Benefits and limitations of each of these tools (in the context of local adaptation)
  * experimental design can be tough
  * links between differential gene expression and trait is unclear

#### Experimental designs to maximize the use of RNA-Seq



### Gene Expression supporting other methods

* Gene expression as both Fst and Qst
* Using gene expression to do population genetics
* using gene expression as a trait, high dimensional trait
  * Cooper et al. 2003
  * Runcie and Mukherjee, 2013

### How gene expression can be used to study local adaptation

* gene expression profiles as a phenotypic trait 
  * Cooper et al.
* incorporating gene expression into findings from genomic data lead to gain in statistical power and add new biological insights 
  * Savolainen et al.
* gene expression as a high dimensional phenotype
* Gene expression can be both Fst and Qst
  * it provides both genetic information and trait information
* Building more complex models by including clustering
  * re-read Thorton paper

* identify novel genetics
* analysis methods - algorithms for clustering, creating modules
* modeling to get biologically meaningful information from gene expression
* study impacts of sequence variation in regulation




### Previous examples and limitations to this approach

* Cooper et al. 2003
* others

Limitations

* differentiating between adaptation and plasticity
  * can gene expression address this? Can this be observed?
  * way to see the difference is genetic information (allele frequencies) over populations/over time from field experiments
