---

## Parallel changes in gene expression after 20,000 generations of evolution in _Escherichia coli_

Cooper, T.F., Rozen, D.E. and Lenski, R.E., 2003. Parallel changes in gene expression after 20,000 generations of evolution in Escherichia coli. Proceedings of the National Academy of Sciences, 100(3), pp.1072-1077.

### Abstract
Twelve populations of Escherichia coli, derived from a common ancestor, evolved in a glucose-limited medium for 20,000 generations. Here we use DNA expression arrays to examine whether gene-expression profiles in two populations evolved in parallel, which would indicate adaptation, and to gain insight into the mechanisms underlying their adaptation. We compared the expression profile of the ancestor to that of clones sampled from both populations after 20,000 generations. The expression of 59 genes had changed significantly in both populations. Remarkably, all 59 were changed in the same direction relative to the ancestor. Many of these genes were members of the cAMP-cAMP receptor protein (CRP) and guanosine tetraphosphate (ppGpp) regulons. Sequencing of several genes controlling the effectors of these regulons found a nonsynonymous mutation in spoT in one population. Moving this mutation into the ancestral background showed that it increased fitness and produced many of the expression changes manifest after 20,000 generations. The same mutation had no effect on fitness when introduced into the other evolved population, indicating that a mutation of similar effect was present already. Our study demonstrates the utility of expression arrays for addressing evolutionary issues including the quantitative measurement of parallel evolution in independent lineages and the identification of beneficial mutations. 

**Key words:** convergent evolution, gene expression, E. coli, ppGpp, macroarray

**Other links:** http://myxo.css.msu.edu/lenski/pdf/1991,%20AmNat,%20Lenski%20et%20al.pdf

### Summary

In this study, the authors were attempting to demonstrate the extent to which phenotypic parallel convergent evolution in _E. coli_ populations extended to the underlying genetics of those populations. The general experiment design was to generate new populations of _E. coli_ from one of two founder populations, then let them grow for 20,000 generations. They measured the fitness is two _E. coli_ cultures competing in a liquid culture, and initial and final densities are calculated by spreading them on plates and distinguishing them by color (Ara+ for white colonies, Ara- form red colonies). They obtained the transcriptional profiles of two of the populations, one from Ara- and one from Ara+. They looked for genes with significantly different expression patterns from the ancestral populations, and found that the two evolved populations were more different from the ancestor than they were from each other. **They had independently developed similar increase in fitness (67% fitness gain) and had similar changes in gene expression.** They found 59 genes with significant differentiation, 12 with increased expression and 47 with decreased expression (relative to the ancestor). **This does not mean that the 59 genes all have separate mutations, not that every genee xpression change is adaptive, but the parallel evolution does imply that there are one or more mutations underlying changes in thee expression that are responsible for the observed increase in fitness.** In the Ara- lineage, they identified a single mutation in _spoT_, a gene that controls the cellular concentration of ppGpp, which is itself broad regulator of multiple genes, that resulted in a single 9.4% fitness gain. A _spoT_ mutation results in changes of expression in 12/59 of the genes identified as having significant gene expression change. They were unable to identify the mutation responsible for the Ara+ fitness gain, but including the Ara- _spoT_ mutation did not convey any fitness gain. There was a _spoT_ mutation in 7 out of the 12 other populations, though all at different locations. **This is parallel evolution extending to the gene level.** They also point out the **utility of using gene-expression prfiles as phenotypic traits.** Cool!

**Questions:**

* how do they know that their strains aren't doing horizontal gene transfer during the fitness assay?
* Since they were unable to identify the Ara+ mutation, does that imply that the mutation was non-genic? And thus, unable to be directly identified with transcriptomic approaches. Or are there big blind spots in macro arrays? I don't know anything about macroarrays.
* This is potentially a nice example of the limit to RNA-Seq approaches - we can only ID within gene mutations, and only see the result of non-genic changes.

Lacking:


## The Molecular Diversity of Adaptive Convergence

> Tenaillon, O., Rodríguez-Verdugo, A., Gaut, R.L., McDonald, P., Bennett, A.F., Long, A.D. and Gaut, B.S., 2012. The molecular diversity of adaptive convergence. Science, 335(6067), pp.457-461.

### Abstract
To estimate the number and diversity of beneficial mutations, we experimentally evolved 115 populations of Escherichia coli to 42.2°C for 2000 generations and sequenced one genome from each population. We identified 1331 total mutations, affecting more than 600 different sites. Few mutations were shared among replicates, but a strong pattern of convergence emerged at the level of genes, operons, and functional complexes. Our experiment uncovered a set of primary functional targets of high temperature, but we estimate that many other beneficial mutations could contribute to similar adaptive outcomes. We inferred the pervasive presence of epistasis among beneficial mutations, which shaped adaptive trajectories into at least two distinct pathways involving mutations either in the RNA polymerase complex or the termination factor rho.

**Key words:**
Convergent evolution, independent mutation

**Lacking:** 
not directly about gene expression

---

## Evaluating the role of natural selection in the evolution of gene regulation

> Fay, J.C. and Wittkopp, P.J., 2008. Evaluating the role of natural selection in the evolution of gene regulation. Heredity, 100(2), pp.191-199.

### Abstract
Surveys of gene expression reveal extensive variability both within and between a wide range of species. Compelling cases have been made for adaptive changes in gene regulation, but the proportion of expression divergence attributable to natural selection remains unclear. Distinguishing adaptive changes driven by positive selection from neutral divergence resulting from mutation and genetic drift is critical for understanding the evolution of gene expression. Here, we review the various methods that have been used to test for signs of selection in genomic expression data. We also discuss properties of regulatory systems relevant to neutral models of gene expression. Despite some potential caveats, published studies provide considerable evidence for adaptive changes in gene expression. Future challenges for studies of regulatory evolution will be to quantify the frequency of adaptive changes, identify the genetic basis of expression divergence and associate changes in gene expression with specific organismal phenotypes.

Key words:
gene expression, evolution, microarray, neutral models, quantitative traits

Lacking:

---

## Predicting Responses to Contemporary Environmental Change Using Evolutionary Response Architectures

Bay, R.A., Rose, N., Barrett, R., Bernatchez, L., Ghalambor, C.K., Lasky, J.R., Brem, R.B., Palumbi, S.R. and Ralph, P., 2017. Predicting responses to contemporary environmental change using evolutionary response architectures. The American Naturalist, 189(5), pp.463-473.

### Abstract
Rapid environmental change currently presents a major threat to global biodiversity and ecosystem functions, and understanding impacts on individual populations is critical to creating reliable predictions and mitigation plans. One emerging tool for this goal is high-throughput sequencing technology, which can now be used to scan the genome for signs of environmental selection in any species and any system. This explosion of data provides a powerful new window into the molecular mechanisms of adaptation, and although there has been some success in using genomic data to predict responses to selection in fields such as agriculture, thus far genomic data are rarely integrated into predictive frameworks of future adaptation in natural populations. Here, we review both theoretical and empirical studies of adaptation to rapid environmental change, focusing on areas where genomic data are poised to contribute to our ability to estimate species and population persistence and adaptation. We advocate for the need to study and model evolutionary response architectures, which integrate spatial information, fitness estimates, and plasticity with genetic architecture. Understanding how these factors contribute to adaptive responses is essential in efforts to predict the responses of species and ecosystems to future environmental change.

Keywords: 
population genomics, environmental change, adaptation plasticity, evolutionary prediction

Lacking:

---

## The genetic differentiation at quantitative trait loci under local adaptation

Le Corre, V. and Kremer, A., 2012. The genetic differentiation at quantitative trait loci under local adaptation. Molecular ecology, 21(7), pp.1548-1566.

### Abstract
Most adaptive traits are controlled by large number of genes that may all together be the targets of selection. Adaptation may thus involve multiple but not necessarily substantial allele frequency changes. This has important consequences for the detection of selected loci and implies that a quantitative genetics framework may be more appropriate than the classical ‘selective sweep’ paradigm. Preferred methods to detect loci involved in local adaptation are based on the detection of ‘outlier’ values of the allelic differentiation FST. A quantitative genetics framework is adopted here to review theoretical expectations for how allelic differentiation at quantitative trait loci (F_{STQ}) relates to (i), neutral genetic differentiation (FST) and (ii), phenotypic differentiation (QST). We identify cases where results of outlier-based methods are likely to be poor and where differentiation at selected loci conveys little information regarding local adaptation. A first case is when neutral differentiation is high, so that local adaptation does not necessitate increased differentiation. A second case is when local adaptation is reached via an increased covariance of allelic effects rather than via allele frequency changes, which is more likely under high gene flow when the number of loci is high and selection is recent. The comparison of theoretical predictions with observed data from the literature suggests that polygenic local adaptation involving only faint allele frequency changes are very likely in some species such as forest trees and for climate-related traits. Recent methodological improvements that may alleviate the weakness of FST-based detection methods are presented.

Keywords: Fst, FSTQ, local adaptation, Qst, quantitative traits, selected loci

Lacking:

---

## Spatiotemporal microbial evolution on antibiotic landscapes

Baym, M., Lieberman, T.D., Kelsic, E.D., Chait, R., Gross, R., Yelin, I. and Kishony, R., 2016. Spatiotemporal microbial evolution on antibiotic landscapes. Science, 353(6304), pp.1147-1151.

### Abstract
A key aspect of bacterial survival is the ability to evolve while migrating across spatially varying environmental challenges. Laboratory experiments, however, often study evolution in well-mixed systems. Here, we introduce an experimental device, the microbial evolution and growth arena (MEGA)–plate, in which bacteria spread and evolved on a large antibiotic landscape (120 × 60 centimeters) that allowed visual observation of mutation and selection in a migrating bacterial front. While resistance increased consistently, multiple coexisting lineages diversified both phenotypically and genotypically. Analyzing mutants at and behind the propagating front, we found that evolution is not always led by the most resistant mutants; highly resistant mutants may be trapped behind more sensitive lineages. The MEGA-plate provides a versatile platform for studying microbial adaption and directly visualizing evolutionary dynamics.

Keywords: 

Lacking:

---

## Limits fo adaptation along environmental gradients

Polechová, J. and Barton, N.H., 2015. Limits to adaptation along environmental gradients. Proceedings of the National Academy of Sciences, 112(20), pp.6401-6406.

### Abstract
Why do species not adapt to ever-wider ranges of conditions, gradually expanding their ecological niche and geographic range? Gene flow across environments has two conflicting effects: although it increases genetic variation, which is a prerequisite for adaptation, gene flow may swamp adaptation to local conditions. In 1956, Haldane proposed that, when the environment varies across space, "swamping" by gene flow creates a positive feedback between low population size and maladaptation, leading to a sharp range margin. However, current deterministic theory shows that, when variance can evolve, there is no such limit. Using simple analytical tools and simulations, we show that genetic drift can generate a sharp margin to a species' range, by reducing genetic variance below the level needed for adaptation to spatially variable conditions. Aided by separation of ecological and evolutionary timescales, the identified effective dimensionless parameters reveal a simple threshold that predicts when adaptation at the range margin fails. Two observable parameters determine the threshold: (i) the effective environmental gradient, which can be measured by the loss of fitness due to dispersal to a different environment; and (ii) the efficacy of selection relative to genetic drift. The theory predicts sharp range margins even in the absence of abrupt changes in the environment. Furthermore, it implies that gradual worsening of conditions across a species' habitat may lead to a sudden range fragmentation, when adaptation to a wide span of conditions within a single species becomes impossible. 

keywords: genetic drift, genetic variation, heterogeneous environments, range margin, species' range

---

## The effect of genetic and environmental variation on metabolic gene expression

Scott, C.P., Williams, D.A. and Crawford, D.L., 2009. The effect of genetic and environmental variation on metabolic gene expression. Molecular ecology, 18(13), pp.2832-2843.

### Abstract
What is the relationship between genetic or environmental variation and the variation in messenger RNA (mRNA) expression? To address this, microarrays were used to examine the effect of genetic and environmental variation on cardiac mRNA expression for metabolic genes in three groups of Fundulus heteroclitus: (i) individuals sampled in the field (field), (ii) field individuals acclimated for 6 months to laboratory conditions (acclimated), or (iii) individuals bred for 10 successive generations in a laboratory environment (G10). The G10 individuals have significantly less genetic variation than individuals obtained in the field and had a significantly lower variation in mRNA expression across all genes in comparison to the other two groups (P = 0.001). When examining the gene specific variation, 22 genes had variation in expression that was significantly different among groups with lower variation in G10 individuals than in acclimated individuals. Additionally, there were fewer genes with significant differences in expression among G10 individuals vs. either acclimated or field individuals: 66 genes have statistically different levels of expression vs. 107 or 97 for acclimated or field groups. Based on the permutation of the data, these differences in the number of genes with significant differences among individuals within a group are unlikely to occur by chance (P < 0.01). Surprisingly, variation in mRNA expression in field individuals is lower than in acclimated individuals. Relative to the variation among individual within a group, few genes have significant differences in expression among groups (seven, 2.3%) and none of these are different between acclimated and field individuals. The results support the concept that genetic variation affects variation in mRNA expression and also suggests that temporal environmental variation associated with estuarine environments does not increase the variation among individuals or add to the differences among groups.

Keywords: Fundulus heteroclitus, microarray, gene expression evolutionary genomics, genetic variation

Lacking:

---

## Topological features of a gene co-expression network predict patterns of natural diversity in environmental response

Des Marais, D.L., Guerrero, R.F., Lasky, J.R. and Scarpino, S.V., 2017, June. Topological features of a gene co-expression network predict patterns of natural diversity in environmental response. In Proc. R. Soc. B (Vol. 284, No. 1856, p. 20170914). The Royal Society.

### Abstract
Molecular interactions affect the evolution of complex traits. For instance, adaptation may be constrained by pleiotropic or epistatic effects, both of which can be reflected in the structure of molecular interaction networks. To date, empirical studies investigating the role of molecular interactions in phenotypic evolution have been idiosyncratic, offering no clear patterns. Here, we investigated the network topology of genes putatively involved in local adaptation to two abiotic stressors—drought and cold—in Arabidopsis thaliana. Our findings suggest that the gene-interaction topologies for both cold and drought stress response are non-random, with genes that show genetic variation in drought expression response (eGxE) being significantly more peripheral and cold response genes being significantly more central than genes which do not show GxE. We suggest that the observed topologies reflect different constraints on the genetic pathways involved in environmental response. The approach presented here may inform predictive models linking genetic variation in molecular signalling networks with phenotypic variation, specifically traits involved in environmental response.

Keywords: gene by environmental interaction, gene regulatory network, Arabidopsis thaliana, abiotic stress, drought, cold

---

## Mathematical and computational analysis of adaptation via Feedback inhibition in signal transduction pathways

Behar, M., Hao, N., Dohlman, H.G. and Elston, T.C., 2007. Mathematical and computational analysis of adaptation via feedback inhibition in signal transduction pathways. Biophysical journal, 93(3), pp.806-821.

### Abstract
We perform a systematic analysis of mechanisms of feedback regulation that underlie short-term adaptation in intracellular signaling systems. Upon receiving an external cue, these systems generate a transient response that quickly returns to basal levels even if the stimulus persists. Signaling pathways capable of short-term adaptation are found in systems as diverse as the high osmolarity response of yeast, gradient sensing in Dictyostelium, and the cytokine response in vertebrates. Using mathematical analysis and computational experiments, we compare different feedback architectures in terms of response amplitude and duration, ability to adapt, and response to variable stimulus levels. Our analysis reveals three important features of these systems: 1), multiple step signaling cascades improve sensitivity to low doses by an effect distinct from signal amplification; 2), some feedback architectures act as signal transducers converting stimulus strength into response duration; and 3), feedback deactivation acts as a dose-dependent switch between transient and sustained responses. Finally, we present characteristic features for each form of feedback regulation that can aid in their identification.

---

## Tradict enable accurate prediction of eukaryotic transcriptional states from 100 marker genes

Biswas, S., Kerner, K., Teixeira, P.J.P.L., Dangl, J.L., Jojic, V. and Wigge, P.A., 2017. Tradict enables accurate prediction of eukaryotic transcriptional states from 100 marker genes. Nature Communications, 8.

### Abstract
Transcript levels are a critical determinant of the proteome and hence cellular function. Because the transcriptome is an outcome of the interactions between genes and their products, it may be accurately represented by a subset of transcript abundances. We develop a method, Tradict (transcriptome predict), capable of learning and using the expression measurements of a small subset of 100 marker genes to predict transcriptome-wide gene abundances and the expression of a comprehensive, but interpretable list of transcriptional programs that represent the major biological processes and pathways of the cell. By analyzing over 23,000 publicly available RNA-Seq data sets, we show that Tradict is robust to noise and accurate. Coupled with targeted RNA sequencing, Tradict may therefore enable simultaneous transcriptome-wide screening and mechanistic investigation at large scales.

Keywords: Bayesian inference, high throughput screeningn, machine learning, transcriptomics

Lacking:

---

## Transcriptional rewiring over evolutionary timescales changes quantitative and qualitative properties of gene expression

Dalal, C.K., Zuleta, I.A., Mitchell, K.F., Andes, D.R., El-Samad, H. and Johnson, A.D., 2016. Transcriptional rewiring over evolutionary timescales changes quantitative and qualitative properties of gene expression. Elife, 5.

### Abstract
Evolutionary changes in transcription networks are an important source of diversity across species, yet the quantitative consequences of network evolution have rarely been studied. Here we consider the transcriptional ‘rewiring’ of the three GAL genes that encode the enzymes needed for cells to convert galactose to glucose. In Saccharomyces cerevisiae, the transcriptional regulator Gal4 binds and activates these genes. In the human pathogen Candida albicans (which last shared a common ancestor with S. cerevisiae some 300 million years ago), we show that different regulators, Rtg1 and Rtg3, activate the three GAL genes. Using single-cell dynamics and RNA-sequencing, we demonstrate that although the overall logic of regulation is the same in both species—the GAL genes are induced by galactose—there are major differences in both the quantitative response of these genes to galactose and in the position of these genes in the overall transcription network structure of the two species.



---

## The Gene Regulatory Landscape of Local Adaptation

Gould, B.A., Chen, Y. and Lowry, D.B., 2017. The Gene Regulatory Landscape of Local Adaptation. bioRxiv, p.145805.

### Abstract
Local adaptation is a key driver of ecological specialization. Despite its importance, the evolution of gene regulatory divergence among locally-adapted populations is poorly understood. Here, we evaluate allele-specific gene expression responses for locally-adapted coastal and inland accessions of the plant Mimulus guttatus in a field reciprocal transplant experiment. We found 18% of transcripts were characterized by habitat-specific differential expression, a pattern consistent with the minimization of fitness trade-offs through expression plasticity. Cis-regulatory variation was pervasive, affecting 92% of differentially expressed genes. Chromosomal inversions were enriched for genes with expression divergence between ecotypes in a habitat-dependent manner, and gene expression heterosis was prevalent across habitats. These results provide multiple new insights into the evolution of transcriptome-wide gene regulatory divergence and plasticity among locally adapted populations.

Lacking: good science

---

## Systematic survey reveals general applicability of "guilt-by-association" within gene coexpression networks

Wolfe, C.J., Kohane, I.S. and Butte, A.J., 2005. Systematic survey reveals general applicability of" guilt-by-association" within gene coexpression networks. BMC bioinformatics, 6(1), p.227.

### Abstract
Biological processes are carried out by coordinated modules of interacting molecules. As clustering methods demonstrate that genes with similar expression display increased likelihood of being associated with a common functional module, networks of coexpressed genes provide one framework for assigning gene function. This has informed the guilt-by-association (GBA) heuristic, widely invoked in functional genomics. Yet although the idea of GBA is accepted, the breadth of GBA applicability is uncertain. We developed methods to systematically explore the breadth of GBA across a large and varied corpus of expression data to answer the following question: To what extent is the GBA heuristic broadly applicable to the transcriptome and conversely how broadly is GBA captured by a priori knowledge represented in the Gene Ontology (GO)? Our study provides an investigation of the functional organization of five coexpression networks using data from three mammalian organisms. Our method calculates a probabilistic score between each gene and each Gene Ontology category that reflects coexpression enrichment of a GO module. For each GO category we use Receiver Operating Curves to assess whether these probabilistic scores reflect GBA. This methodology applied to five different coexpression networks demonstrates that the signature of guilt-by-association is ubiquitous and reproducible and that the GBA heuristic is broadly applicable across the population of nine hundred Gene Ontology categories. We also demonstrate the existence of highly reproducible patterns of coexpression between some pairs of GO categories.

---

## An Expanded View of Complex Traits: From Polygenic to Omnigenic

Boyle, E.A., Li, Y.I. and Pritchard, J.K., 2017. An Expanded View of Complex Traits: From Polygenic to Omnigenic. Cell, 169(7), pp.1177-1186.

### Abstract
A central goal of genetics is to understand the links between genetic variation and disease. Intuitively, one might expect disease-causing variants to cluster into key pathways that drive disease etiology. But for complex traits, association signals tend to be spread across most of the genome—including near many genes without an obvious connection to disease. We propose that gene regulatory networks are sufficiently interconnected such that all genes expressed in disease-relevant cells are liable to affect the functions of core disease-related genes and that most heritability can be explained by effects on genes outside core pathways. We refer to this hypothesis as an ‘‘omnigenic’’ model.

---

## Signatures of local adaptation in lowland and highland teosintes from whole‐genome sequencing of pooled samples

Fustier, M.A., Brandenburg, J.T., Boitard, S., Lapeyronnie, J., Eguiarte, L.E., Vigouroux, Y., Manicacci, D. and Tenaillon, M.I., 2017. Signatures of local adaptation in lowland and highland teosintes from whole‐genome sequencing of pooled samples. Molecular Ecology, 26(10), pp.2738-2756.

### Abstract
Spatially varying selection triggers differential adaptation of local populations. Here, we mined the determinants of local adaptation at the genomewide scale in the two closest maize wild relatives, the teosintes Zea mays ssp parviglumis and ssp. mexicana. We sequenced 120 individuals from six populations: two lowland, two intermediate and two highland populations sampled along two altitudinal gradients. We detected 8,479,581 single nucleotide polymorphisms (SNPs) covered in the six populations with an average sequencing depth per site per population ranging from 17.03 to 32.23. Population diversity varied from 0.10 to 0.15, and linkage disequilibrium decayed very rapidly. We combined two differentiation-based methods, and correlation of allele frequencies with environmental variables to detect outlier SNPs. Outlier SNPs displayed significant clustering. From clusters, we identified 47 candidate regions. We further modified a haplotype-based method to incorporate genotype uncertainties in haplotype calling, and applied it to candidate regions. We retrieved evidence for selection at the haplotype level in 53% of our candidate regions, and in 70% of the cases the same haplotype was selected in the two lowland or the two highland populations. We recovered a candidate region located within a previously characterized inversion on chromosome 1. We found evidence of a soft sweep at a locus involved in leaf macrohair variation. Finally, our results revealed frequent colocalization between our candidate regions and loci involved in the variation of traits associated with plant–soil interactions such as root morphology, aluminium and low phosphorus tolerance. Soil therefore appears to be a major driver of local adaptation in teosintes.

---

## Extensive Sex-Specific Nonadditivity of Gene Expression in _Drosophila melanogaster_

Gibson, G., Riley-Berger, R., Harshman, L., Kopp, A., Vacha, S., Nuzhdin, S. and Wayne, M., 2004. Extensive sex-specific nonadditivity of gene expression in Drosophila melanogaster. Genetics, 167(4), pp.1791-1799.

### Abstract
Assessment of the degree to which gene expression is additive and heritable has important implications for understanding the maintenance of variation, adaptation, phenotypic divergence, and the mapping of genotype onto phenotype. We used whole-genome transcript profiling using Agilent long-oligonucleotide microarrays representing 12,017 genes to demonstrate that gene transcription is pervasively nonadditive in Drosophila melanogaster. Comparison of adults of two isogenic lines and their reciprocal F1 hybrids revealed 5820 genes as significantly different between at least two of the four genotypes in either males or females or across both sexes. Strikingly, while 25% of all genes differ between the two parents, 33% differ between both F1’s and the parents, averaged across sexes. However, only 5% of genes show overdominance, suggesting that heterosis for expression is rare.

---

## 
Genetic signals of origin, spread, and introgression in a large sample of maize landraces

Van Heerwaarden, J., Doebley, J., Briggs, W.H., Glaubitz, J.C., Goodman, M.M., Gonzalez, J.D.J.S. and Ross-Ibarra, J., 2011. Genetic signals of origin, spread, and introgression in a large sample of maize landraces. Proceedings of the National Academy of Sciences, 108(3), pp.1088-1092.

### Abstract
The last two decades have seen important advances in our knowledge of maize domestication, thanks in part to the contributions of genetic data. Genetic studies have provided firm evidence that maize was domesticated from Balsas teosinte (Zea mays subspecies parviglumis), a wild relative that is endemic to the mid- to lowland regions of southwestern Mexico. An interesting paradox remains, however: Maize cultivars that are most closely related to Balsas teosinte are found mainly in the Mexican highlands where subspecies parviglumis does not grow. Genetic data thus point to primary diffusion of domesticated maize from the highlands rather than from the region of initial domestication. Recent archeological evidence for early lowland cultivation has been consistent with the genetics of domestication, leaving the issue of the ancestral position of highland maize unresolved. We used a new SNP dataset scored in a large number of accessions of both teosinte and maize to take a second look at the geography of the earliest cultivated maize. We found that gene flow between maize and its wild relatives meaningfully impacts our inference of geographic origins. By analyzing differentiation from inferred ancestral gene frequencies, we obtained results that are fully consistent with current ecological, archeological, and genetic data concerning the geography of early maize cultivation.

---

## The role of gene expression in ecological speciation

Pavey, S.A., Collin, H., Nosil, P. and Rogers, S.M., 2010. The role of gene expression in ecological speciation. Annals of the New York Academy of Sciences, 1206(1), pp.110-129.

### Abstract
Ecological speciation is the process by which barriers to gene flow between populations evolve due to adaptive divergence via natural selection. A relatively unexplored area in ecological speciation is the role of gene expression. Gene expression may be associated with ecologically important phenotypes not evident from morphology and play a role during colonization of new environments. Here we review two potential roles of gene expression in ecological speciation: (1) its indirect role in facilitating population persistence and (2) its direct role in contributing to genetically based reproductive isolation. We find indirect evidence that gene expression facilitates population persistence, but direct tests are lacking. We also find clear examples of gene expression having effects on phenotypic traits and adaptive genetic divergence, but links to the evolution of reproductive isolation itself remain indirect. Gene expression during adaptive divergence seems to often involve complex genetic architectures controlled by gene networks, regulatory regions, and “eQTL hotspots.” Nonetheless, we review how approaches for isolating the functional mutations contributing to adaptive divergence are proving to be successful. The study of gene expression has promise for increasing our understanding ecological speciation, particularly when integrative approaches are applied.

---

## 
The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish

Reid, N.M., Proestou, D.A., Clark, B.W., Warren, W.C., Colbourne, J.K., Shaw, J.R., Karchner, S.I., Hahn, M.E., Nacci, D., Oleksiak, M.F. and Crawford, D.L., 2016. The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science, 354(6317), pp.1305-1308.

### Abstract
Atlantic killifish populations have rapidly adapted to normally lethal levels of pollution in four urban estuaries. Through analysis of 384 whole killifish genome sequences and comparative transcriptomics in four pairs of sensitive and tolerant populations, we identify the aryl hydrocarbon receptor–based signaling pathway as a shared target of selection. This suggests evolutionary constraint on adaptive solutions to complex toxicant mixtures at each site. However, distinct molecular variants apparently contribute to adaptive pathway modification among tolerant populations. Selection also targets other toxicity-mediating genes and genes of connected signaling pathways; this indicates complex tolerance phenotypes and potentially compensatory adaptations. Molecular changes are consistent with selection on standing genetic variation. In killifish, high nucleotide diversity has likely been a crucial substrate for selective sweeps to propel rapid adaptation.

---

## The relationship between phenotypic and environmental variation: do physiological responses reduce interindividual differences?

Oleksiak, M.F. and Crawford, D.L., 2012. The relationship between phenotypic and environmental variation: do physiological responses reduce interindividual differences?. Physiological and Biochemical Zoology, 85(6), pp.572-584.

### Abstract
What is the effect of a variable environment on phenotypic variation? Does the physiological response to a new environment increase or decrease the differences among individuals? We provide a speculative hypothesis suggesting that the induction of a physiological response to environmental change minimizes phenotypic differences among individuals in outbred genetically variable populations. Although this suggestion runs counter to the general idea that environmental variation induces phenotypic variation, we provide evidence that this is not always the case. One explanation for this counterintuitive hypothesis is that in a variable environment, the physiological mechanism that maintains homeostasis changes the concentrations of active transcription factors (TFs). This change in TFs reduces the effectiveness of nucleotide polymorphisms in TF binding sites and thus reduces the variation among individuals in mRNA expression and in the phenotypes affected by these mRNA transcripts. Thus, there are fewer differences among individuals in a variable environment compared with the variation observed in a constant environment. Our conjecture is that the physiological mechanisms that maintain homeostasis in response to environmental variation canalize phenotypic variation. If our hypothesis is correct, then the physiological canalization of gene expression in a variable environment hides genetic variation and thereby reduces the evolutionary costs of polymorphism. This hypothesis provides a new perspective on the mechanisms by which high levels of genetic variation can persist in real-world populations.

---

## A model for the genetic architecture of quantitative traits under stabilizing selection

Simons, Y.B., Bullaughey, K., Hudson, R.R. and Sella, G., 2017. A model for the genetic architecture of quantitative traits under stabilizing selection. arXiv preprint arXiv:1704.06707.

### Abstract
Genome-wide association studies (GWAS) in humans are revealing the genetic architecture of biomedical, life history and anthropomorphic traits, i.e., the frequencies and effect sizes of variants contributing to heritable variation in a trait. To interpret these findings, we need to understand how genetic architecture is shaped by basic population genetics processes—notably, by mutation, natural selection and genetic drift. Because many quantitative traits are subject to stabilizing selection and genetic variation that affects one trait often affects many others, we model the genetic architecture of a focal trait that arises under stabilizing selection in a multi-dimensional trait space. We solve the model for the phenotypic distribution and allelic dynamics at steady state and derive robust, closed form solutions for summaries of genetic architecture. Our results suggest that the distribution of genetic variance among the loci discovered in GWAS take a simple form that depends on one evolutionary parameter, and provide a simple interpretation for missing heritability and why it varies among traits. We test our predictions against the results of GWAS for height and body mass index (BMI) and find that they fit the data well, allowing us to make inferences about the degree of pleiotropy and the mutational target size. Our findings help to understand why GWAS for height explain more of the heritable variance than similarlysized GWAS for BMI, and to predict how future increases in sample size will translate into explained heritability.

---

## Reevaluation of SNP heritability in complex human traits

Speed, D., Cai, N., Johnson, M.R., Nejentsev, S., Balding, D.J. and UCLEB Consortium, 2017. Reevaluation of SNP heritability in complex human traits. Nature genetics.

### Abstract
SNP heritability, the proportion of phenotypic variance explained by SNPs, has been reported for many hundreds of traits. Its estimation requires strong prior assumptions about the distribution of heritability across the genome, but current assumptions have not been thoroughly tested. By analyzing imputed data for a large number of human traits, we empirically derive a model that more accurately describes how heritability varies with minor allele frequency (MAF), linkage disequilibrium (LD) and genotype certainty. Across 19 traits, our improved model leads to estimates of common SNP heritability on average 43% (s.d. 3%) higher than those obtained from the widely used software GCTA and 25% (s.d. 2%) higher than those from the recently proposed extension GCTA-LDMS. Previously, DNase I hypersensitivity sites were reported to explain 79% of SNP heritability; using our improved heritability model, their estimated contribution is only 24%.

---

## Genetic diversity in a crop metapopulation

Van Heerwaarden, J., Van Eeuwijk, F.A. and Ross-Ibarra, J., 2010. Genetic diversity in a crop metapopulation. Heredity, 104(1), pp.28-39.

### Abstract
The need to protect crop genetic resources has sparked a growing interest in the genetic diversity maintained in traditional farming systems worldwide. Although traditional seed management has been proposed as an important determinant of genetic diversity and structure in crops, no models exist that can adequately describe the genetic effects of seed management. We present a metapopulation model that accounts for several features unique to managed crop populations. Using traditional maize agriculture as an example, we develop a coalescence-based model of a crop metapopulation undergoing pollen and seed flow as well as seed replacement. In contrast to metapopulation work on natural systems, we model seed migration as episodic and originating from a single source per population rather than as a constant immigration from the entire metapopulation. We find that the correlated origin of migrants leads to surprising results, including a loss of invariance of within-deme diversity and a parabolic relationship between FST and migration quantity. In contrast, the effects of migration frequency on diversity and structure are more similar to classical predictions, suggesting that seed migration in managed crop populations cannot be described by a single parameter. In addition to migration, we investigate the effects of deme size and extinction rates on genetic structure, and show that high levels of pollen migration may mask the effects of seed management on structure. Our results highlight the importance of analytically evaluating the effects of deviations from classical metapopulation models, especially in systems for which data are available to estimate specific model parameters.

---

## 
The biological importance of measuring individual variation

Crawford, D.L. and Oleksiak, M.F., 2007. The biological importance of measuring individual variation. Journal of Experimental Biology, 210(9), pp.1613-1621.

### Abstract
Functional genomics research using Fundulus heteroclitus has focused on variation among individuals because of the evolutionary importance and value of Fundulus in explaining the human condition (why individual humans are different and are affected differently by stress, disease and drugs). Among different populations and species of Fundulus, there are evolutionarily adaptive differences in gene expression. This natural variation in gene expression seems to affect cardiac metabolism because up to 81% of the variation in glucose utilization observed in isolated heart ventricles is related to specific patterns of gene expression. The surprising result from this research is that among different groups of individuals, the expression of mRNA from different metabolic pathways explains substrate-specific metabolism. For example, variation in oxidative phosphorylation mRNAs explains glucose metabolism for one group of individuals but expression of glucose metabolism genes explains this metabolism in a different group of individuals. This variation among individuals has important implications for studies using inbred strains: conclusions based on one individual or one strain will not necessarily reflect a generalized conclusion for a population or species. Finally, there are surprisingly strong positive and negative correlations among metabolic genes, both within and between pathways. These data suggest that measures of mRNA expression are meaningful, yet there is a complexity in how gene expression is related to physiological processes.

---

## Phenotypic plasticity in gene expression contributes to divergence of locally adapted populations of Fundulus heteroclitus

Dayan, D.I., Crawford, D.L. and Oleksiak, M.F., 2015. Phenotypic plasticity in gene expression contributes to divergence of locally adapted populations of Fundulus heteroclitus. Molecular Ecology, 24(13), pp.3345-3359.

### Abstract
We examine the interaction between phenotypic plasticity and evolutionary adaptation using muscle gene expression levels among populations of the fish Fundulus heteroclitus acclimated to three temperatures. Our analysis reveals shared patterns of phenotypic plasticity due to thermal acclimation as well as non-neutral patterns of variation among populations adapted to different thermal environments. For the majority of significant differences in gene expression levels, phenotypic plasticity and adaptation operate on different suites of genes. The subset of genes that demonstrate both adaptive differences and phenotypic plasticity, however, exhibit countergradient variation of expression. Thus, expression differences among populations counteract environmental effects, reducing the phenotypic differentiation between populations. Finally, gene-by-environment interactions among genes with non-neutral patterns of expression suggest that the penetrance of adaptive variation depends on the environmental conditions experienced by the individual.

---

## Using environmental correlations to identify loci underlying local adaptation

Coop, G., Witonsky, D., Di Rienzo, A. and Pritchard, J.K., 2010. Using environmental correlations to identify loci underlying local adaptation. Genetics, 185(4), pp.1411-1423.

### Abstract
Loci involved in local adaptation can potentially be identified by an unusual correlation between allele frequencies and important ecological variables or by extreme allele frequency differences between geographic regions. However, such comparisons are complicated by differences in sample sizes and the neutral correlation of allele frequencies across populations due to shared history and gene flow. To overcome these difficulties, we have developed a Bayesian method that estimates the empirical pattern of covariance in allele frequencies between populations from a set of markers and then uses this as a null model for a test at individual SNPs. In our model the sample frequencies of an allele across populations are drawn from a set of underlying population frequencies; a transform of these population frequencies is assumed to follow a multivariate normal distribution. We first estimate the covariance matrix of this multivariate normal across loci using a Monte Carlo Markov chain. At each SNP, we then provide a measure of the support, a Bayes factor, for a model where an environmental variable has a linear effect on the transformed allele frequencies compared to a model given by the covariance matrix alone. This test is shown through power simulations to outperform existing correlation tests. We also demonstrate that our method can be used to identify SNPs with unusually large allele frequency differentiation and offers a powerful alternative to tests based on pairwise or global FST. Software is available at http://www.eve.ucdavis.edu/gmcoop/.

---

## Adaptation to elevated CO2 in different biodiversity contexts

Kleynhans, E.J., Otto, S.P., Reich, P.B. and Vellend, M., 2016. Adaptation to elevated CO2 in different biodiversity contexts. Nature communications, 7.

### Abstract
In the absence of migration, species persistence depends on adaption to a changing environment, but whether and how adaptation to global change is altered by community diversity is not understood. Community diversity may prevent, enhance or alter how species adapt to changing conditions by influencing population sizes, genetic diversity and/or the fitness landscape experienced by focal species. We tested the impact of community diversity on adaptation by performing a reciprocal transplant experiment on grasses that evolved for 14 years under ambient and elevated CO2, in communities of low or high species richness. Using biomass as a fitness proxy, we find evidence for local adaptation to elevated CO2, but only for plants assayed in a community of similar diversity to the one experienced during the period of selection. Our results indicate that the biological community shapes the very nature of the fitness landscape within which species evolve in response to elevated CO2.

---

## Too much of a good thing: the unique and repeated paths toward copper adaptation

Gerstein, A.C., Ono, J., Lo, D.S., Campbell, M.L., Kuzmin, A. and Otto, S.P., 2015. Too much of a good thing: the unique and repeated paths toward copper adaptation. Genetics, 199(2), pp.555-571.

### Abstract
Copper is a micronutrient essential for growth due to its role as a cofactor in enzymes involved in respiration, defense against oxidative damage, and iron uptake. Yet too much of a good thing can be lethal, and yeast cells typically do not have tolerance to copper levels much beyond the concentration in their ancestral environment. Here, we report a short-term evolutionary study of Saccharomyces cerevisiae exposed to levels of copper sulfate that are inhibitory to the initial strain. We isolated and identified adaptive mutations soon after they arose, reducing the number of neutral mutations, to determine the first genetic steps that yeast take when adapting to copper. We analyzed 34 such strains through whole-genome sequencing and by assaying fitness within different environments; we also isolated a subset of mutations through tetrad analysis of four lines. We identified a multilayered evolutionary response. In total, 57 single base-pair mutations were identified across the 34 lines. In addition, gene amplification of the copper metallothionein protein, CUP1-1, was rampant, as was chromosomal aneuploidy. Four other genes received multiple, independent mutations in different lines (the vacuolar transporter genes VTC1 and VTC4; the plasma membrane H+-ATPase PMA1; and MAM3, a protein required for normal mitochondrial morphology). Analyses indicated that mutations in all four genes, as well as CUP1-1 copy number, contributed significantly to explaining variation in copper tolerance. Our study thus finds that evolution takes both common and less trodden pathways toward evolving tolerance to an essential, but highly toxic, micronutrient.

---

## Haploids adapt faster than diploids across a range of environments

Gerstein, A.C., Cleathero, L.A., Mandegar, M.A. and Otto, S.P., 2011. Haploids adapt faster than diploids across a range of environments. Journal of evolutionary biology, 24(3), pp.531-540.

### Abstract
Despite a great deal of theoretical attention, we have limited empirical data about how ploidy influences the rate of adaptation. We evolved isogenic haploid and diploid populations of Saccharomyces cerevisiae for 200 generations in seven different environments. We measured the competitive fitness of all ancestral and evolved lines against a common competitor and find that in all seven environments, haploid lines adapted faster than diploids, significantly so in three environments. We apply theory that relates the rates of adaptation and measured effective population sizes to the properties of beneficial mutations. We obtained rough estimates of the average selection coefficients in haploids between 2% and 10% for these first selected mutations. Results were consistent with semi-dominant to dominant mutations in four environments and recessive to additive mutations in two other environments. These results are consistent with theory that predicts haploids should evolve faster than diploids at large population sizes.

---

## Connecting genomic patterns of local adaptation and niche suitability in teosintes

Aguirre‐Liguori, J.A., Tenaillon, M.I., Vázquez‐Lobo, A., Gaut, B.S., Jaramillo‐Correa, J.P., Montes‐Hernandez, S., Souza, V. and Eguiarte, L.E., 2017. Connecting genomic patterns of local adaptation and niche suitability in teosintes. Molecular Ecology.

### Abstract
The central-abundance hypothesis predicts that local adaptation is a function of the distance to the center of a species’ geographic range. To test this hypothesis, we gathered genomic diversity data from 49 populations, 646 individuals and 33,464 SNPs of two wild relatives of maize, the teosintes Zea mays ssp. parviglumis and Zea. mays. ssp. mexicana. We examined the association between the distance to their climatic and geographic centroids and the enrichment of SNPs bearing signals of adaptation. We identified candidate adaptive SNPs in each population by combining neutrality tests and cline analyzes. By applying linear regression models, we found that the number of candidate SNPs is positively associated with niche suitability, while genetic diversity is reduced at the limits of the geographic distribution. Our results suggest that overall, populations located at the limit of the species’ niches are adapting locally. We argue that local adaptation to this limit could initiate ecological speciation processes and facilitate adaptation to global change.

---

## Genetics of global gene expression

Rockman, M.V. and Kruglyak, L., 2006. Genetics of global gene expression. Nature Reviews Genetics, 7(11), pp.862-872.

### Abstract
A new field of genetic analysis of global gene expression has emerged in recent years, driven by the realization that traditional techniques of linkage and association analysis can be applied to thousands of transcript levels measured by microarrays. Genetic dissection of transcript abundance has shed light on the architecture of quantitative traits, provided a new approach for connecting DNA sequence variation with phenotypic variation, and improved our understanding of transcriptional regulation and regulatory variation.

---

## Understanding mechanisms underlying human gene expression variation with RNA sequencing

Pickrell, J.K., Marioni, J.C., Pai, A.A., Degner, J.F., Engelhardt, B.E., Nkadori, E., Veyrieras, J.B., Stephens, M., Gilad, Y. and Pritchard, J.K., 2010. Understanding mechanisms underlying human gene expression variation with RNA sequencing. Nature, 464(7289), pp.768-772.

### Abstract
Understanding the genetic mechanisms underlying natural variation in gene expression is a central goal of both medical and evolutionary genetics, and studies of expression quantitative trait loci (eQTLs) have become an important tool for achieving this goal1. Although all eQTL studies so far have assayed messenger RNA levels using expression microarrays, recent advances in RNA sequencing enable the analysis of transcript variation at unprecedented resolution. We sequenced RNA from 69 lymphoblastoid cell lines derived from unrelated Nigerian individuals that have been extensively genotyped by the International HapMap Project2. By pooling data from all individuals, we generated a map of the transcriptional landscape of these cells, identifying extensive use of unannotated untranslated regions and more than 100 new putative protein-coding exons. Using the genotypes from the HapMap project, we identified more than a thousand genes at which genetic variation influences overall expression levels or splicing. We demonstrate that eQTLs near genes generally act by a mechanism involving allele-specific expression, and that variation that influences the inclusion of an exon is enriched within and near the consensus splice sites. Our results illustrate the power of high-throughput sequencing for the joint analysis of variation in transcription, splicing and allele-specific expression across individuals.

---

## Polygenic evolution of a sugar specialization trade-off in yeast

Roop, J.I., Chang, K.C. and Brem, R.B., 2016. Polygenic evolution of a sugar specialization trade-off in yeast. Nature, 530(7590), pp.336-339.

### Abstract
The evolution of novel traits can involve many mutations scattered throughout the genome. Detecting and validating such a suite of alleles, particularly if they arose long ago, remains a key challenge in evolutionary genetics. Here we dissect an evolutionary trade-off of unprecedented genetic complexity between long-diverged species. When cultured in 1% glucose medium supplemented with galactose, Saccharomyces cerevisiae, but not S. bayanus or other Saccharomyces species, delayed commitment to galactose metabolism until glucose was exhausted. Promoters of seven galactose (GAL) metabolic genes from S. cerevisiae, when introduced together into S. bayanus, largely recapitulated the delay phenotype in 1% glucose–galactose medium, and most had partial effects when tested in isolation. Variation in GAL coding regions also contributed to the delay when tested individually in 1% glucose–galactose medium. When combined, S. cerevisiae GAL coding regions gave rise to profound growth defects in the S. bayanus background. In medium containing 2.5% glucose supplemented with galactose, wild-type S. cerevisiae repressed GAL gene expression and had a robust growth advantage relative to S. bayanus; transgenesis of S. cerevisiae GAL promoter alleles or GAL coding regions was sufficient for partial reconstruction of these phenotypes. S. cerevisiae GAL genes thus encode a regulatory program of slow induction and avid repression, and a fitness detriment during the glucose–galactose transition but a benefit when glucose is in excess. Together, these results make clear that genetic mapping of complex phenotypes is within reach, even in deeply diverged species.

---

## The fitness consequences of aneuploidy are driven by condition-dependent gene effects

Sunshine, A.B., Payen, C., Ong, G.T., Liachko, I., Tan, K.M. and Dunham, M.J., 2015. The fitness consequences of aneuploidy are driven by condition-dependent gene effects. PLoS biology, 13(5), p.e1002155.

## Abstract
Aneuploidy is a hallmark of tumor cells, and yet the precise relationship between aneuploidy and a cell’s proliferative ability, or cellular fitness, has remained elusive. In this study, we have combined a detailed analysis of aneuploid clones isolated from laboratory-evolved populations of Saccharomyces cerevisiae with a systematic, genome-wide screen for the fitness effects of telomeric amplifications to address the relationship between aneuploidy and cellular fitness. We found that aneuploid clones rise to high population frequencies in nutrient-limited evolution experiments and show increased fitness relative to wild type. Direct competition experiments confirmed that three out of four aneuploid events isolated from evolved populations were themselves sufficient to improve fitness. To expand the scope beyond this small number of exemplars, we created a genome-wide collection of >1,800 diploid yeast strains, each containing a different telomeric amplicon (Tamp), ranging in size from 0.4 to 1,000 kb. Using pooled competition experiments in nutrient-limited chemostats followed by high-throughput sequencing of strain-identifying barcodes, we determined the fitness effects of these >1,800 Tamps under three different conditions. Our data revealed that the fitness landscape explored by telomeric amplifications is much broader than that explored by single-gene amplifications. As also observed in the evolved clones, we found the fitness effects of most Tamps to be condition specific, with a minority showing common effects in all three conditions. By integrating our data with previous work that examined the fitness effects of single-gene amplifications genome-wide, we found that a small number of genes within each Tamp are centrally responsible for each Tamp’s fitness effects. Our genome-wide Tamp screen confirmed that telomeric amplifications identified in laboratory-evolved populations generally increased fitness. Our results show that Tamps are mutations that produce large, typically condition-dependent changes in fitness that are important drivers of increased fitness in asexually evolving populations.


---

## The landscape of genetic complexity across 5,700 gene expression traits in yeast

Brem, R.B. and Kruglyak, L., 2005. The landscape of genetic complexity across 5,700 gene expression traits in yeast. Proceedings of the National Academy of Sciences of the United States of America, 102(5), pp.1572-1577.

### Abstract
Many studies have identified quantitative trait loci (QTLs) that contribute to continuous variation in heritable traits of interest. However, general principles regarding the distribution of QTL numbers, effect sizes, and combined effects of multiple QTLs remain to be elucidated. Here, we characterize complex genetics underlying inheritance of thousands of transcript levels in a cross between two strains of Saccharomyces cerevisiae. Most detected QTLs have weak effects, with a median variance explained of 27% for highly heritable transcripts. Despite the high statistical power of the study, no QTLs were detected for 40% of highly heritable transcripts, indicating extensive genetic complexity. Modeling of QTL detection showed that only 3% of highly heritable transcripts are consistent with single-locus inheritance, 17–18% are consistent with control by one or two loci, and half require more than five loci under additive models. Strikingly, analysis of parent and progeny trait distributions showed that a majority of transcripts exhibit transgressive segregation. Sixteen percent of highly heritable transcripts exhibit evidence of interacting loci. Our results will aid design of future QTL mapping studies and may shed light on the evolution of quantitative traits. 

---

## Gene–environment interaction in yeast gene expression

Smith, E.N. and Kruglyak, L., 2008. Gene–environment interaction in yeast gene expression. PLoS biology, 6(4), p.e83.

### Abstract
The effects of genetic variants on phenotypic traits often depend on environmental and physiological conditions, but such gene–environment interactions are poorly understood. Recently developed approaches that treat transcript abundances of thousands of genes as quantitative traits offer the opportunity to broadly characterize the architecture of gene–environment interactions. We examined the genetic and molecular basis of variation in gene expression between two yeast strains (BY and RM) grown in two different conditions (glucose and ethanol as carbon sources). We observed that most transcripts vary by strain and condition, with 2,996, 3,448, and 2,037 transcripts showing significant strain, condition, and strain–condition interaction effects, respectively. We expression profiled over 100 segregants derived from a cross between BY and RM in both growth conditions, and identified 1,555 linkages for 1,382 transcripts that show significant gene–environment interaction. At the locus level, local linkages, which usually correspond to polymorphisms in cis-regulatory elements, tend to be more stable across conditions, such that they are more likely to show the same effect or the same direction of effect across conditions. Distant linkages, which usually correspond to polymorphisms influencing trans-acting factors, are more condition-dependent, and often show effects in different directions in the two conditions. We characterized a locus that influences expression of many growth-related transcripts, and showed that the majority of the variation is explained by polymorphism in the gene IRA2. The RM allele of IRA2 appears to inhibit Ras/PKA signaling more strongly than the BY allele, and has undergone a change in selective pressure. Our results provide a broad overview of the genetic architecture of gene–environment interactions, as well as a detailed molecular example, and lead to key insights into how the effects of different classes of regulatory variants are modulated by the environment. These observations will guide the design of studies aimed at understanding the genetic basis of complex traits.

---

## The role of regulatory variation in complex traits and disease

Albert, F.W. and Kruglyak, L., 2015. The role of regulatory variation in complex traits and disease. Nature Reviews Genetics, 16(4), pp.197-212.

### Abstract
We are in a phase of unprecedented progress in identifying genetic loci that cause variation in traits ranging from growth and fitness in simple organisms to disease in humans. However, a mechanistic understanding of how these loci influence traits is lacking for the majority of loci. Studies of the genetics of gene expression have emerged as a key tool for linking DNA sequence variation to phenotypes. Here, we review recent insights into the molecular nature of regulatory variants and describe their influence on the transcriptome and the proteome. We discuss conceptual advances from studies in model organisms and present examples of complete chains of causality that link individual polymorphisms to changes in gene expression, which in turn result in physiological changes and, ultimately, disease risk.
